### ISSUE TITLE
No attribute found "UserId" 

### ISSUE THREAD

**Issue by yashdargude** [adrianhajdin/healthcare#19]
while register from new mail it throws error that userId is not defined?
do help

An error occurred while retrieving the patient details: AppwriteException [Error]: Invalid query: Attribute not found in schema: userId
    at Client.call (webpack-internal:///(rsc)/./node_modules/node-appwrite/lib/client.js:206:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Databases.listDocuments (webpack-internal:///(rsc)/./node_modules/node-appwrite/lib/services/databases.js:1658:16)
    at async $$ACTION_3 (webpack-internal:///(rsc)/./lib/actions/patient.actions.ts:80:26)
    at async Register (webpack-internal:///(rsc)/./app/patients/[userId]/register/page.tsx:31:21) {
  code: 400,
  type: 'general_query_invalid',
  response: {
    message: 'Invalid query: Attribute not found in schema: userId',
    code: 400,
    type: 'general_query_invalid',
    version: '1.5.7'
  }

**Comment by bharatpaliwal-169** [comment]
Hey @yashdargude kindly check you `attributes` in your database schema in appwrite. Remember to check the spell `userId` or `UserID` whatever you are using also check all other params there in schema. Once that you can follow a little bit further in the video and you will know the solution by yourself.

**Comment by liandab** [comment]
can i fix this issues?

**Comment by adekmo** [comment]
> Hey @yashdargude kindly check you `attributes` in your database schema in appwrite. Remember to check the spell `userId` or `UserID` whatever you are using also check all other params there in schema. Once that you can follow a little bit further in the video and you will know the solution by yourself.

hei, i have this issue,
Invalid document structure: Missing required attribute "user_id"
![image](https://github.com/user-attachments/assets/41ce45d2-3cf3-4b2b-82a6-18b6cbfcc21b)


![image](https://github.com/user-attachments/assets/0adddfb5-2052-478f-8fae-e980587d17dc)

how to fixed it?
thnks

**Comment by Sharmin097** [comment]
@adekmo  did you find any solution of your problem? i am also having this issue

**Comment by adekmo** [comment]
Yes i did bro. The problem with the code. I write userId in the code, not user_id.

**Comment by Sharmin097** [comment]
Thanks for your response

**Comment by theTechbug** [comment]
The issue 'No attribute found: UserId' seems to be due to a typo, either in the Appwrite configuration or in the code itself. Please verify the spelling of 'UserId' in both places to ensure consistency."
